docs/en/sql-reference/statements/create/user.md

```diff
@@ -144,6 +144,12 @@ In ClickHouse Cloud, by default, passwords must meet the following complexity re
 
     The available password types are: `plaintext_password`, `sha256_password`, `double_sha1_password`.
 
+7. Multiple authentication methods can be specified: 
+
+   ```sql
+   CREATE USER user1 IDENTIFIED WITH plaintext_password by '1' ADD IDENTIFIED WITH bcrypt_password by '2' ADD IDENTIFIED WITH plaintext_password by '3''
```

Comment:
Syntax with ADD looks odd here.

Using verb 'add' in the middle of CREATE USER ... is strange. Maybe just comma separated?

Reply:
ALTER ... ADD looks ok. 

But CREATE ... ADD - is weird.

Reply:
@pufit I have discussed in private with @filimonov about the syntax and we thought it would be better to simplify it to a comma-separated list like the below:

`CREATE USER user1 IDENTIFIED WITH plaintext_password by '1', sha256_password by '2', bcypt_password by '3'...`
`ALTER USER user1 IDENTIFIED WITH plaintext_password by '1', sha256_password by '2', bcypt_password by '3'...`
`ALTER USER user1 ADD IDENTIFIED WITH plaintext_password by '1', sha256_password by '2', bcypt_password by '3'...`

As you probably know, the `WITH` keyword can be omitted and the authentication type does not need to be specified. In this case, I opted for the following:

`CREATE USER user1 IDENTIFIED by '1', plaintext_password by '2', by '3'` -- If the default auth type is sha256, then the user has two sha256 auth methods and one plaintext.

@alexey-milovidov fyi